### PR TITLE
[bitnami/fluentd] Add possibility to mount extra volumes

### LINKED ISSUE #2339: [bitnami/fluentd] Unable to mount extra volumes
**Which chart**:
[bitnami/fluentd]

**Describe the bug**
The chart is currently missing the possibility to mount extra volumes inside Fluentd forwarders.
This is needed for example when you need to monitor `systemd' modules and the system store logs outside `/var/log/` path (e.g CentOS, Debian).

**Expected behavior**
Ability to specify extra options in `values` and `values-production` files to mount extra volumes.

**Version of Helm and Kubernetes**:

- Output of `helm version`:

```
version.BuildInfo{Version:"v3.1.2", GitCommit:"d878d4d45863e42fd5cff6743294a11d28a9abce", GitTreeState:"clean", GoVersion:"go1.13.8"}
```

- Output of `kubectl version`:

```
Client Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.0", GitCommit:"9e991415386e4cf155a24b1da15becaa390438d8", GitTreeState:"clean", BuildDate:"2020-03-25T14:58:59Z", GoVersion:"go1.13.8", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.0", GitCommit:"9e991415386e4cf155a24b1da15becaa390438d8", GitTreeState:"clean", BuildDate:"2020-03-25T14:50:46Z", GoVersion:"go1.13.8", Compiler:"gc", Platform:"linux/amd64"}

```

**Additional context**

### PR DESCRIPTION
**Pull request by irizzant**
Signed-off-by: Ivan Rizzante <ivan.rizzante@tech.sdb.it>

**Description of the change**

This PR allows users to mount extra volumes in Fluentd forwarders so that they can use extra host folders as source for their logs.

**Benefits**

Users will be able to mount extra host folders to use as logs source.

**Possible drawbacks**

**Applicable issues**

  - fixes #2339

**Additional information**

**Checklist** 
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [X] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/fluentd/README.md [hunk 1]
```diff
@@ -135,6 +135,8 @@ The following tables lists the configurable parameters of the kibana chart and t
...
 | `aggregator.tolerations`                        | Tolerations for pod assignment                                                                                 | `[]`                                                                                                    |
 | `aggregator.affinity`                           | Affinity for pod assignment                                                                                    | `{}`                                                                                                    |
+| `aggregator.extraVolumes`                       | Extra volumes                                                                                                  | `nil`                                                                                                    |
+| `aggregator.extraVolumeMounts`                  | Mount extra volume(s),                                                                                         | `nil`                                                                                                    |
```
Comment (carrodher) [inline bitnami/fluentd/README.md:139]: Shouldn't it be placed in the forwarder section as `forwarder.*`?

**Comment by carrodher** [comment]
Thanks for your contribution, I see you are adding this feature to the forwarder (in the values and in the template) but in the README you are talking about aggregator. Once fixed this "typo" I will proceed to launch the PR in our testing & release pipeline.

File: bitnami/fluentd/README.md (hunk 1 above)
Comment (irizzant) [reply bitnami/fluentd/README.md]: Yes I fixed the typo

**Comment by carrodher** [comment]
I just added a commit replacing the name of the variable itself apart from the position in the README

**Comment by irizzant** [comment]
Thank you

**Pull request merged**